### DISCUSSION TITLE
Empty Custom Core Pstates

### DISCUSSION THREAD

**Discussion by Fetnak** [DavidS95/Smokeless_UMAF#29] [Q&A] [answered]
I receive empty menu when I go to:
Device Manager - AMD CBS - CPU Common Options - Performace - Custom Core Pstates - Accept
Is there any secret? Or just Pstates are locked by HP?

Laptop: HP 15s-eq2000
CPU: Ryzen 5 5500U

**Comment by enesuzun2002** [comment]
check the Device Manager - AMD CBS - NBIO - SMU page there should be the cppc settings

**Reply by sammcj** [reply]
I was looking for this too, but the tool crashes when entering the NBIO menu?

**Reply by Crandel** [reply]
I was able to enable amd-pstate support on linux on Asus TUF A15 notebook with AMD ryzen 2 4800 CPU.  Thank you very much!

**Reply by enesuzun2002** [reply]
@Crandel how is it working currently?

**Reply by Crandel** [reply]
I bought recently another laptop, but old one still working

**Reply by enesuzun2002** [reply]
did it improve battery life, that is what i am actually concerned about

**Reply by LFRon** [reply]
But I cannot found SMU Page in NBIO at Ryzen 7 8845HS Laptop

**Reply by chris1892006** [reply]
> I was able to enable amd-pstate support on linux on Asus TUF A15 notebook with AMD ryzen 2 4800 CPU. Thank you very much!

How, please?  I have DELL G5 SE 5505, 4800H CPU, same as you.  We unlocked everything in the BIOS except for "Custom Core Pstates".  I accept & nothing.  By the way, XFR common options is pretty sweet for PBO control.

**Reply by Crandel** [reply]
I don't remember how exactly. I'm using another laptop at the moment

**Reply by chris1892006** [reply]
@Crandel May you advise on if you opened the Section_PE32_image_CbsSetupDxeRN_CbsSetupDxeRN.sct.0.0.en-US.uefi.ifr.txt & searched for "Custom Core Pstates"?  I'm trying to understand how these bios modders unlocked the options.  Can you compare these differences?  Modded on LEFT, Untouched on RIGHT.
<img width="1920" height="343" alt="notepad_RFzr6lXu3R" src="https://github.com/user-attachments/assets/656fbd4e-8162-4715-bbd4-3ca671bb6cd2" />

**Comment by Fetnak** [comment, marked as answer]
I found a solution. I sold this laptop :D
